packages/calypso-products/src/plans-list.tsx

```diff
@@ -868,6 +875,10 @@ const getPlanPersonalDetails = (): IncompleteWPcomPlan => ( {
 
 		let features = baseFeatures;
 
+		if ( isGlobalStylesGridChangesVariation() ) {
+			features = [ ...features, FEATURE_WORDADS, FEATURE_PAYMENT_TRANSACTION_FEES_8 ];
```

Comment:
`FEATURE_WORDADS` should be removed from here too.
```suggestion
			features = [ ...features, FEATURE_PAYMENT_TRANSACTION_FEES_8 ];
```

Reply:
done